dataflow/gemma/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This uses Ubuntu with Python 3.11
+ARG SERVING_BUILD_IMAGE=tensorflow/tensorflow:latest-gpu
+
+FROM ${SERVING_BUILD_IMAGE}
+
+WORKDIR /workspace
+
+RUN apt-get update -y
```

Comment:
PTAL at https://docs.docker.com/develop/develop-images/instructions/#run . In this case merging RUN commands is better. Some discussion about that on https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-which-is-better .

Reply:
interesting. thank you!